app/models/lead.rb

```diff
@@ -1,5 +1,5 @@
 class Lead <ApplicationRecord
-  validates :salesforce_id, presence: true, uniqueness: true
+  #validates :salesforce_id, presence: true, uniqueness: true
```

Comment:
why did this get commented out?

Reply:
oh, because we don't have one at creation.. can we keep the uniqueness validation though?

Reply:
uniqueness is back